sdk/search/search-documents/samples/v11/javascript/setup.js

```diff
@@ -7,14 +7,14 @@
 
 const { KnownAnalyzerNames } = require("@azure/search-documents");
 
-export const WAIT_TIME = 4000;
+module.exports.WAIT_TIME = 4000;
```

Comment:
was this manually updated?

Reply:
Will we be manually updating this whenever we run `dev-tool samples pubish` command?
Any better solution for such scripts, @witemple-msft?

Reply:
yes